DevelopGitHub/GitHubExtension/GitHubExtension.WebUI/FrontEnd/.idea/.name

```diff
@@ -0,0 +1 @@
+GitHubExtension
```

Comment:
Please do not push .idea folder to repository (add it to .gitignore file in the frontend folder)